app/javascript/utilities/__tests__/timeAgo.test.js

```diff
@@ -0,0 +1,129 @@
+import '../../../assets/javascripts/utilities/timeAgo';
+
+/* global globalThis timeAgo */
+
+describe('timeAgo', () => {
+  afterAll(() => {
+    delete globalThis.timeAgo;
```

Comment:
Since the script `'../../../assets/javascripts/utilities/timeAgo'` gets imported, but does not export anything, only add `timeAgo` to `globalThis`, we need to remove it after the tests run.

Maybe overkill.